pytorch_lightning/strategies/launchers/subprocess_script_hydra.py

```diff
@@ -0,0 +1,82 @@
+import os
```

Comment:
I propose to move this to subprocess_script.py for the reason outlined in my other comment.